core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -109,9 +118,18 @@ func IsRollupWithL1Support(chainType config.ChainType) bool {
 }
 
 func NewL1GasOracle(lggr logger.Logger, ethClient ethClient, chainType config.ChainType) L1Oracle {
+	var priceReader daPriceReader
+	if chainType == config.ChainOptimismBedrock || chainType == config.ChainKroma {
+		priceReader = newOPPriceReader(lggr, ethClient, chainType, KromaGasOracleAddress)
```

Comment:
The precompile addresses were hidden in the diff so I almost missed this. I believe the OP and Kroma gas oracles have different precompile addresses. `0x420000000000000000000000000000000000000F` vs `0x4200000000000000000000000000000000000005`. Think you'll need to split this into 2 separate if statements